.github/workflow/integrate.yml

```diff
@@ -0,0 +1,21 @@
+name: Java Continuous Integration
+
+on:
```

Comment:
I would set this `on: [ push, pull_request ]`, so that way everything runs through it